superset/assets/src/visualizations/Table/Table.css

```diff
@@ -29,3 +29,7 @@ table.table thead th.sorting:after, table.table thead th.sorting_asc:after, tabl
 .like-pre {
   white-space: pre-wrap;
 }
+
+.selected-row {
+  background-color:#ADD8E6 !important;
+}
```

Comment:
is #ADD8E6 color is finalized by ux if it is not configurable.

Reply:
Nice catch. We will need this configurable as we are a self service platform. However sync with @jollyparamjit  to get the color as per current theme

Reply:
is there a way to provide this with more specificity css rule rather than `!important`

Reply:
`!important` changes are not working fine with existing framework css implementation, so will pick this as a separate task.